docs/extensions/life-cycle-events/index.mdx

```diff
@@ -235,22 +235,22 @@ function waitForRequest(method: string, url: string) {
   let requestId = ''
 
   return new Promise((resolve, reject) => {
-    server.events.on('request:start', (req) => {
```

Comment:
Life-cycle events are exposed under `.events` property on both worker and server instances since [0.35.0](https://github.com/mswjs/msw/releases/tag/v0.35.0). The current usage examples are correct.

Note that our docs don't support multi-versioning, and always document the latest version of the library (that you should be using at all times).

Reply:
Could you please revert this change?